axe_selenium_python/tests/test_axe.py

```diff
@@ -5,33 +5,30 @@
 from os import path
 
 import pytest
+from selenium import webdriver
 
+from ..axe import Axe
 
-@pytest.mark.nondestructive
-def test_execute(axe):
-    """Run axe against base_url and verify JSON output."""
-    axe.inject()
-    data = axe.execute()
-    assert data is not None, data
+_DEFAULT_TEST_FILE = path.join(path.dirname(__file__), 'test_page.html')
 
 
-@pytest.mark.nondestructive
-def test_report(axe):
-    """Test that report exists."""
-    axe.inject()
-    results = axe.execute()
-    violations = results["violations"]
-    report = axe.report(violations)
-    assert report is not None, report
+@pytest.fixture(
```

Comment:
Move this into 1 line, I also think you can shorten it by removing the url and just noting the issue number. xfailing Chrome denotes an issue with Chrome. I like the verbosity, but in this instance I think it's not needed.

`@pytest.fixture(params=('Firefox', pytest.mark.xfail('Chrome', reason="see issue #118")))`